### PR TITLE
Term testing

### PR DESCRIPTION
**Pull request by BWitemeyer**

### CODE REVIEW

**Comment by BWitemeyer** [comment]
NOTE: This PR changes some things that are also changed in other PRs, so it may be best to merge this one after those.

**Review by adauzat90 — APPROVED** [review]
Nothing to really add the code looks good.

**Review by henrywang1 — APPROVED** [review]
Nicely done.

File: zlp-scheduler/db/seeds.rb [hunk 1]
```diff
@@ -92,10 +92,8 @@ def create_schedule(student,schedule_name, classes)
         @user.password = "Temp"
         @user.cohort_id = @cohort.id
         @user.save
-<<<<<<< HEAD
        
-=======
```
Comment (adauzat90) [inline zlp-scheduler/db/seeds.rb:97]: Need to fix this might need to recreate the PR if you already have

**Review by adauzat90 — CHANGES_REQUESTED** [review]

**Pull request merged**